src/docfx/lib/collections/WorkQueue.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Docs.Build
+{
+    internal class WorkQueue<T>
```

Comment:
>WorkQueue [](start = 19, length = 9)

with this, do we still need Parallel.ForEach? Can we only keep one?

Reply:
nop, removed